### PR TITLE
update to oep-2

### PR DESCRIPTION
**Pull request by awaisdar001**
Update OEP-2 tags. 

[PROD-440](https://openedx.atlassian.net/browse/PROD-440) / [PROD-1012](https://openedx.atlassian.net/browse/PROD-1012)

### CODE REVIEW

File: openedx.yaml [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 nick: desg
+tags:
+  - webservice
 oeps:
   oep-2: True
   oep-7: True
+  oep-10: True
   oep-18: True
   oep-19: True
 openedx-release: {ref: master}
```
Comment (nedbat) [inline openedx.yaml]: @adamstankiewicz @brittneyexline Should this repo be tagged as part of Open edX releases? I would have thought no.

Comment (brittneyexline) [reply openedx.yaml]: are you asking if this should be made available as part of the next official openedx release? given the current use case as well as the ambiguity of ownership or future development here, i would tend to say this isn't something that would be useful or relevant to the open source community at this point.

Comment (nedbat) [reply openedx.yaml]: I would agree.  So this line needs to be removed.

Comment (awaisdar001) [reply openedx.yaml]: Sure, I can remove it. thanks for the context.

Comment (awaisdar001) [reply openedx.yaml]: Updated.

**Comment by awaisdar001** [comment]
I have updated the PR. 
@adamstankiewicz / @schenedx  can you review, please?

File: openedx.yaml [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+supporting_teams:
+  - masters-devs
+  - edx/enterprise
 nick: desg
+tags:
+  - webservice
 oeps:
   oep-2: True
   oep-7: True
+  oep-10: True
```
Comment (iloveagent57) [inline openedx.yaml]: I think you want to delete this line, too, if it's not meant to be part of an Open edX release.

Comment (iloveagent57) [reply openedx.yaml]: ...or maybe set it to `False`?

**Comment by kdmccormick** [comment]
@schenedx  Are we taking over Designer now?

File: openedx.yaml [hunk 3]
```diff
@@ -1,13 +1,16 @@
 # This file describes this Open edX repo, as described in OEP-2:
 # http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
 
-owner: tuchfarber
```
Comment (schenedx) [inline openedx.yaml]: I thought we should pin the `owner` to a team instead of a person. How about `Masters and Micromasters Squad`?

Comment (schenedx) [reply openedx.yaml]: Or simply `masters-devs`

Comment (awaisdar001) [reply openedx.yaml]: `masters-devs` sounds good

**Comment by awaisdar001** [comment]
PR is updated; fyi -- @iloveagent57 / @schenedx

File: openedx.yaml [hunk 4]
```diff
@@ -1,13 +1,18 @@
...
-supporting_teams: edx/enterprise
+owner: masters-devs
+supporting_teams:
+  - masters-devs
```
Comment (schenedx) [inline openedx.yaml]: If you have the owner to be `masters-devs`, you don't need that team here.

**Comment by awaisdar001** [comment]
@iloveagent57 please review. thanks.

**Pull request merged**